DAO.sol

```diff
@@ -513,7 +524,7 @@ contract DAO is DAOInterface, Token, TokenSale {
             lastTimeMinQuorumMet = now;
             rewardToken[address(this)] += p.amount;
             totalRewardToken += p.amount;
-        } else if (quorum >= minQuorum(p.amount) && p.nay >= p.yea) {
+        } else if (quorum >= minQuorum(p.amount) && p.nay >= p.yea || !updateContractQuroumCheck) {
```

Comment:
What happens if the quorum checks out and `updateContractQuorumCheck = true` and we get here because people downvoted the proposal? Isn't the proposal deposit lost?